vagrant-kubernetes-istio/setup_kubectl_config_host.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 
 # Setting up kubectl on host to talk to kubernetest cluster on Vagrant VM.
+
+# Unset KUBECONFIG in case users set it point to a k8s cluster
+unset KUBECONFIG
```

Comment:
Should this value be saved before unsetting, so that we it can be reset in cleanup_host script?

Something like this:
export  KUBECONFIG_SAVED=$KUBECONFIG
https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#set-the-kubeconfig-environment-variable